.bacon.yml

```diff
@@ -104,5 +104,5 @@ test_suites:
       sort_order: '1'
       timeout: '60'
       script_name: e2e-monolith
-      criteria: MERGE
+      criteria: OPTIONAL
```

Comment:
@shawyu-okta @glenfannin-okta  
I was getting consistent `e2e-monolith` failure with error 
`Failed to find monolith_tool artifact: 2023.05.2-begin-88-g728587241fcb`. So i made this test optional.  Is this correct?

Reply:
Making this optional is fine, we still have some work to do to make the dockolith test suite more stable. cc: @jaredperreault-okta